src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1029,6 +1029,29 @@ public function upsert(array $values, $uniqueBy, $update = null)
         );
     }
 
+    /**
+     * Update the column's update timestamp.
+     *
+     * @param  string|null  $column
+     * @return int|false
+     */
+    public function touch($column = null)
+    {
+        $time = $this->model->freshTimestamp();
+
+        if ($column) {
+            return $this->toBase()->update([$column => $time]);
```

Comment:
Is it intentional that you can touch a column if the model `usesTimestamps()` returns false?

Reply:
Yes:

https://github.com/laravel/framework/blob/fdf3ae9760729652673359d3e2cced727ba25b92/src/Illuminate/Database/Eloquent/Concerns/HasTimestamps.php#L22-L37